src/lib/LBT/LBT.cpp

```diff
@@ -34,17 +37,30 @@ uint32_t ICACHE_RAM_ATTR SpreadingFactorToRSSIvalidDelayUs(SX1280_RadioLoRaSprea
   // SF7 ~218us (60us + SF7 symbol time)
   // SF9 ~218us (Odd one out, measured to same as SF7 wait time)
 
-  switch(SF)
-    {
-      case SX1280_LORA_SF5: return 100;
-      case SX1280_LORA_SF6: return 141;
-      case SX1280_LORA_SF7: return 218;
-      case SX1280_LORA_SF9: return 218;
-      default: return 218; // Values above 100mW are not relevant, default to 100mW threshold
-    }
+  if (radio_type == RADIO_TYPE_SX128x_LORA)
+  {
+    switch(SF)
+      {
+        case SX1280_LORA_SF5: return 100;
+        case SX1280_LORA_SF6: return 141;
+        case SX1280_LORA_SF7: return 218;
+        case SX1280_LORA_SF8: return 218;
+        default: return 218;
+      }
+  }
+  else if (radio_type == RADIO_TYPE_SX128x_FLRC)
+  {
+    // case SX1280_FLRC_BT_1: 
+    return 60 + 20; // switching time + 20us settling time (seems fine when testing)
```

Comment:
Worthwhile adding a `case` here so that we dont get caught out if we change the RF parameters in the future?

Reply:
I'll remove the comment as stated above, it's not necessary.